config/env_sample.json

```diff
@@ -1,14 +1,26 @@
 {
     "LOG_LEVEL": "WARNING",
-    "/*  Fill in values for database host, port, name, user, and password!": "*/",
-    "UDW": {
-        "type": "PostgreSQL",
-        "params": {
-            "host": "",
-            "port": "",
-            "dbname": "",
-            "user": "",
-            "password": ""
+    "/*  Fill in values for database host, port, name, user, and password for each data source!": "*/",
+    "DATA_SOURCES" : {
+        "UDW": {
+            "type": "PostgreSQL",
+            "params": {
+                "host": "",
+                "port": "",
+                "dbname": "",
+                "user": "",
+                "password": ""
+            }
+        },
+        "UDP Context Store": {
```

Comment:
Do we have a generic UDP Credentials or special user created for using it in validation script? As I remember everybody got their own credentials for logging into the UDP. We don't want to give our own credentials  on openshift, not does it matter but i fell it is good to keep a seperation.